_posts/2018-06-26-why-you-need-https.md

```diff
@@ -0,0 +1,159 @@
+
+### HTTPS is required for the modern web
+
+One of the more promising developments of the last few years, is the pervasiveness and effectiveness of Progressive Web Apps (PWAs). PWAs is the name coined for a set of technologies that provide a feature-set for mobile browsing akin to native applications, yet is entirely served through the web browser. PWAs require all communication to be done over HTTPS. Some of the possibilities with PWAs that were previously relegated to native applications only are:
+
+- Providing content and services based on the user's location data
+- Providing interaction with the user's camera and microphone within the browsing experience
+- Sending push notifications
+- Serving off-line content
+
+If you aren't taking advantage of any of these features that are possible through PWAs, it's something your organization should strongly consider to further engage users. Before the ambitions to be on feature parity with native applications are fully borne-out, PWAs will continue to evolve the power of layering deeper engagement with users on top of your existing mobile experience with minimal effort. PWAs simply do not work over HTTP. HTTPS is required to open the door to their possibilities.
+
+## Barriers to HTTPS have been lifted
+
+Historically, considering a move to HTTPS has been held back by some valid concerns for webmasters whose job it was to select where and how their websites were hosted. A few of the fundamental apprehensions could be categorized as:
+
+- No perceived benefit. People often assumed if they weren't collecting financial or personal information, it wasn't necessary.
+- Performance costs. We know reducing latency is crucial for optimizing conversions and HTTPS does require additional communication and computation which could slow sites down.
+- Financial costs. HTTPS was just too complex and costly to implement for some.
+
+As for the first, I believe we've covered why holding this belief currently is a misconception. With regard to performance, with the adoption of the HTTP/2 protocol, HTTPS [usually outperforms HTTP](http://www.httpvshttps.com/). On the financial front, large strides have been made across many hosting platforms who bundle HTTPS into their offerings for free. This coupled with the [Let's Encrypt](https://letsencrypt.org/) certificate authority first offering free certificates (which they still do) and next making it easy to automatically renew those certificates has greatly reduced the cost of implementation.
```

Comment:
Instead of including these refutations to the points above in a paragraph, I'd recommend including them within the bullet points, e.g.

- No perceived benefit. People often assumed if they weren’t collecting financial or personal information, it wasn’t necessary. I believe we've covered why holding this belief is a misconception
- Performance costs. We know reducing latency is crucial for optimizing conversions and HTTPS does require additional communication and computation which could slow sites down (**note**: previously slowed sites down?). However, with the adoption of the HTTP/2 protocol, HTTPS usually outperforms HTTP.
- Financial costs. HTTPS was just too complex and costly to implement for some. Large strides have been made across many hosting platforms who bundle HTTPS into their offerings for free. This coupled with the Let’s Encrypt certificate authority first offering free certificates (which they still do) and next making it easy to automatically renew those certificates has greatly reduced the cost of implementation.